qa/test_template.sh

```diff
@@ -60,8 +60,11 @@ do
 
             # If we just installed tensorflow, we need to reinstall DALI TF plugin
             if [[ "$inst" == *tensorflow* ]]; then
-                # The package name can be nvidia-dali-tf-plugin,  nvidia-dali-tf-plugin-weekly or  nvidia-dali-tf-plugin-nightly
+                # The package name can be nvidia-dali-tf-plugin,  nvidia-dali-tf-plugin-weekly or nvidia-dali-tf-plugin-nightly
+                # Different DALI can be installed as a dependency of nvidia-dali so uninstall it too
                 pip uninstall -y `pip list | grep nvidia-dali-tf-plugin | cut -d " " -f1` || true
+                pip uninstall -y `pip list | grep nvidia-dali | cut -d " " -f1` || true
```

Comment:
Actually you don't need the previous line now

Reply:
I thought about it and I think it may be better to leave it here to be more explicit. One day we may want to remove just this line.